### PR TITLE
SW-5885 Removed feedback from review. Updated application statuses to show in review

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#3116** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3116?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/Application/ApplicationCard.tsx [hunk 1]
```diff
@@ -33,36 +33,39 @@ export default function ApplicationCard({
...
+        case 'Submitted':
+        case 'In Review':
+          return strings.IN_REVIEW;
+        case 'Waitlist':
+        case 'Issue Active':
+        case 'Issue Pending':
+        case 'Issue Resolved':
+          return strings.IN_REVIEW;
+        case 'Accepted':
+          return strings.IN_REVIEW;
```
Comment (constanzauanini) [inline src/components/Application/ApplicationCard.tsx:62]: Shoud this string be ACCEPTED?

File: src/components/Application/ApplicationCard.tsx [hunk 2]
```diff
@@ -33,36 +33,39 @@ export default function ApplicationCard({
...
+        case 'Not Accepted':
+          return strings.IN_REVIEW;
```
Comment (constanzauanini) [inline src/components/Application/ApplicationCard.tsx:64]: Same here, for not aceepted

File: src/components/Application/ApplicationCard.tsx (hunk 1 above)
Comment (tommylau523) [reply src/components/Application/ApplicationCard.tsx:62]: No we want to show In Reivew for every result statuses

File: src/components/Application/ApplicationCard.tsx (hunk 2 above)
Comment (tommylau523) [reply src/components/Application/ApplicationCard.tsx:64]: No we want to show In Reivew for every result statuses

**Review by constanzauanini — APPROVED** [review]

**Pull request merged**